internal/venice-jdk-compatibility-test/build.gradle

```diff
@@ -0,0 +1,25 @@
+plugins {
```

Comment:
Let's use 2 spaces per tab like in other gradle files.

Reply:
Sure. Thanks for the catch. Will fix